pkg/frame/config.go

```diff
@@ -0,0 +1,22 @@
+package frame
+
+func (config *Config) VisibleHeight() int {
```

Comment:
I really like moving all the height logic into one file/type.  We learned that its easy to lose track of when we add or subtract one, and I think this should mitigate mistakes in the future